src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -18,27 +18,38 @@ namespace trace {
 
 CorProfiler* profiler = nullptr;
 
-CorProfiler::CorProfiler() { Info("CorProfiler::CorProfiler"); }
+CorProfiler::CorProfiler() { Debug("CorProfiler::CorProfiler"); }
 
 HRESULT STDMETHODCALLTYPE
 CorProfiler::Initialize(IUnknown* cor_profiler_info_unknown) {
-  is_attached_ = FALSE;
-  Info("CorProfiler::Initialize");
-  Info("Environment variables:");
-
-  WSTRING env_vars[] = {environment::tracing_enabled,
-                        environment::debug_enabled,
-                        environment::integrations_path,
-                        environment::process_names,
-                        environment::agent_host,
-                        environment::agent_port,
-                        environment::env,
-                        environment::service_name,
-                        environment::disabled_integrations,
-                        environment::clr_disable_optimizations};
-
-  for (auto&& env_var : env_vars) {
-    Info("  ", env_var, "=", GetEnvironmentValue(env_var));
+  // check if debug mode is enabled
+  const auto debug_enabled_value =
+      GetEnvironmentValue(environment::debug_enabled);
+
+  if (debug_enabled_value == "1"_W || debug_enabled_value == "true"_W) {
+    debug_logging_enabled = true;
+    Debug("Debug mode enabled in ", environment::debug_enabled);
+  }
+
+  Debug("CorProfiler::Initialize");
+
+  if (debug_logging_enabled) {
+    Debug("Environment variables:");
```

Comment:
I feel like we should leave the environment variables as Info calls since they control the profiler behavior and there are only a limited number of variables to print.

Reply:
Ok, reverted this change in 3ba359aeee5c7cdc4dcb09331ad9001b02b6040b.